media/css/plugincheck/plugincheck.less

```diff
@@ -15,22 +23,64 @@
         }
     }
     .preamble {
-        .font-size(24px);
-        letter-spacing: -0.25px;
         margin-bottom: 0;
+        max-width: 700px;
         .open-sans-light;
+        .font-size(24px);
         font-weight: normal;
-        margin-bottom:0;
-        line-height: 100%;
+        line-height: 1.2;
+        letter-spacing: -0.25px;
         text-shadow: 0px 1px 0px rgba(255, 255, 255, 0.75);
         color: rgb(72, 72, 72);
+
+        @media only screen and (max-width: @breakDesktop) {
+            max-width: 500px;
+            .font-size(1.2rem);
+        }
+
+        @media only screen and (max-width: @breakDesktop) {
+            max-width: 500px;
+            .font-size(1.2rem);
+        }
     }
-}
 
-html[dir="rtl"] {
-    #main-feature > .download-button {
-        right: auto;
-        left: 0;
+    .firefox-logo {
+        float: left;
+        margin-right: 30px;
+        width: 145px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            width: 115px;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            float: none;
+            display: block;
+            margin: 0 auto;
+            width: 95px;
+        }
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        .large {
+            .font-size(3.5rem);
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        .large {
+            .font-size(3rem);
+        }
+
+        img[data-high-res] {
```

Comment:
What is this selector for, is it perhaps overly-specific and could apply to more than what we intend?